spec.bs

```diff
@@ -7047,11 +7075,24 @@ The following algorithm will be added to the [[FETCH#fetching]] section:
       that is, scripts making the {{WindowOrWorkerGlobalScope/fetch()}} request aren't able to load
       the header value.
     1. [=list/For each=] |bid| of |additionalBids|:
-      1. Let |nonceAndAdditionalBid| be the result of [=strictly splitting=] |bid| on U+003A (:).
-      1. If |nonceAndAdditionalBid|'s [=list/size=] is not 2, then [=iteration/continue=].
-      1. Let |nonce| be |nonceAndAdditionalBid|[0].
-      1. If |nonce|'s [=string/length=] is not 36, then [=iteration/continue=].
-      1. Set |storedAdditionalBidsHeaders|[|nonce|] to |nonceAndAdditionalBid|[1].
+      1. Let |parts| be the result of [=strictly splitting=] |bid| on U+003A (:).
+      1. Let |bidWithMetadata| be a new [=signed additional bid with metadata=].
+      1. If |parts|'s [=list/size=] is 3:
+        1. Let |auctionNonce| be |parts|[0].
+        1. If |auctionNonce|'s [=string/length=] is not 36, then [=iteration/continue=].
+        1. Let |sellerNonce| be |parts|[1].
+        1. If |sellerNonce|'s [=string/length=] is not 36, then [=iteration/continue=].
+        1. Let |bidWithMetadata|'s [=signed additional bid with metadata/seller nonce=] be
+          |sellerNonce|.
+        1. Let |bidWithMetadata|'s [=signed additional bid with metadata/signed additional bid=] be
+          |parts|[2].
+        1. Set |storedAdditionalBidsHeaders|[|auctionNonce|] to |bidWithMetadata|.

```

Comment:
storedAdditionalBidsHeaders is a `traversable navigable/captured ad auction additional bids headers`, which line 6887 says "has a `captured ad auction additional bids headers`, which is a `map` whose `keys` are `auction nonces` and whose `values` are `lists` of `signed additional bid with metadata`."

As such, I assume this should be `[=list/Append=]` instead of `Set`?